core/src/test/java/io/grpc/internal/AutoConfiguredLoadBalancerFactoryTest.java

```diff
@@ -17,7 +17,6 @@
 package io.grpc.internal;
 
 import static com.google.common.truth.Truth.assertThat;
-import static io.grpc.LoadBalancer.ATTR_LOAD_BALANCING_CONFIG;
```

Comment:
Checks for `ATTR_LOAD_BALANCING_CONFIG` should be preserved, because it's still there for back-ward compatibility.